exercises/pangram/pangram_test.rb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'pangram'
+
+# Test data version:
+# 180638f Merge pull request #217 from ErikSchierboom/patch-2
+
+class PangramTest < Minitest::Test
+  # Below we use `%Q` syntax to create strings.
+  # This let's us worry less about escaping quotes.
+  # You can read more about Ruby's string literal syntax here:
+  # http://ruby-doc.org/core-2.3.0/doc/syntax/literals_rdoc.html#label-Strings
+
+  def test_sentence_empty
+    str = %Q()
```

Comment:
Not appropriate here, you are just creating a string.  This should be str = '' or String.new